packages/flutter/test/material/text_selection_theme_test.dart

```diff
@@ -69,52 +71,12 @@ void main() {
     await tester.pumpAndSettle();
     final EditableTextState editableTextState = tester.firstState(find.byType(EditableText));
     final RenderEditable renderEditable = editableTextState.renderEditable;
```

Comment:
Why has this test and the next one been hollowed out?

Reply:
Before we had two tests for defaults: one with opt-in turned off and one for opt-in turn on. With this PR there is only one default (the opt-in) case, so this replaced both with just the opt-in case.